sympy/stats/crv_types.py

```diff
@@ -2627,6 +2629,88 @@ def Pareto(name, xm, alpha):
 
     return rv(name, ParetoDistribution, (xm, alpha))
 
+#-------------------------------------------------------------------------------
+# PERT distribution ----------------------------------------------------------
+
+
+class PERTDistribution(SingleContinuousDistribution):
+    _argnames = ('a', 'b', 'c')
+
+    @property
+    def set(self):
+        return Interval(self.a, self.c)
+
+    @staticmethod
+    def check(a, b, c):
+        _value_check((b > a, b < c),
```

Comment:
There should also be a value check for `a < c`.
It is possible for the current check to not raise if it gets (a, b, c) as (2, Symbol('b'), 1).

Reply:
Done. Thanks!